runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java

```diff
@@ -242,6 +243,14 @@ public boolean matches(Client client, TileObject object)
 	WEISS_OBSTACLES(68, "Shortcut", null, ObjectID.MY2ARM_CLIFF_SHORTCUT_1, ObjectID.MY2ARM_CLIFF_SHORTCUT_2, ObjectID.MY2ARM_CLIFF_SHORTCUT_3, ObjectID.MY2ARM_CLIFF_SHORTCUT_3_ROPETRAIL_MULTI, ObjectID.MY2ARM_CLIFF_SHORTCUT_4, ObjectID.MY2ARM_CLIFF_SHORTCUT_5, ObjectID.MY2ARM_CLIFF_SHORTCUT_6),
 	WEISS_FARMING_PATCH_BOULDER(0, "Shortcut", null, ObjectID.MY2ARM_HERBPATCH_ACCESS),
 	ARCEUUS_ESSENSE_NORTH(69, "Rock Climb", new WorldPoint(1759, 3873, 0), ObjectID.ARCHEUUS_RUNESTONE_SHORTCUT_GREY_SHORTCUT_NORTH),
+	DARKMEYER_WALL_ROCKS(69, "Rocks", new WorldPoint(3563, 3380, 0), ObjectID.HH_MASTER005, ObjectID.DARKM_WALL_ROCK_SHORTCUT)
```

Comment:
why is the stash unit listed as an obstacle here?

Reply:
Because it is. Added on [last week's patch](https://secure.runescape.com/m=news/more-doom-tweaks-poll-84--summer-sweep-up-changes?oldschool=1):

> With level 69 Agility, you can now jump over some of the rocks found on top of the walls of Darkmeyer to access the Castle Drakan master clue step there. These rocks are accessible by climbing the Darkmeyer wall shortcut and running north-west.
> - You can also right-click the inconspicuous rocks/S.T.A.S.H units to jump back, should you need to! However, you cannot use that shortcut without completing Sins of the Father, and without first unlocking the Darkmeyer wall shortcut with a long rope.